src/_base/harness/attributes/common.yml

```diff
@@ -218,9 +218,9 @@ attributes.default:
       config:
         agent_socket: "tcp://blackfire:8707"
     ext-tideways:
-      enable: no
+      enable: "= ('tideways' in @('app.services') ? 'tideways' : '')"
```

Comment:
```suggestion
      enable: "= @('services.tideways.enabled')"
```

Reply:
Is services.tideways.enabled suitable for pipeline usage?

Reply:
`app.services` isn't, but see my other comment about using helm .Values.docker.services.tideways to control that